src/components/harvestLogs/HarvestLogDrawer.tsx

```diff
@@ -428,13 +447,67 @@ const HarvestLogDrawer = ({
           Cancel
         </Button>
 
-        <Button
-          variant="contained"
-          onClick={handleSubmit(onSubmit)}
-          disabled={isLoading}
-        >
+        <Button variant="contained" onClick={handleOpen} disabled={isLoading}>
           Save
         </Button>
+
+        <Dialog open={open} onClose={handleClose}>
+          <div
+            style={{
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "center",
+              justifyContent: "center",
+              padding: "40px 24px 28px 24px",
+              borderRadius: "16px",
+            }}
+          >
+            <TelegramLogo size={40} style={{ color: "#055E40" }} />
```

Comment:
we can have a chat and review how to access the color palette

Reply:
On this icon, when I use the primary option, it seems like it's not working. I have changed it to where it does work